ivy/functional/backends/jax/activations.py

```diff
@@ -37,7 +37,7 @@ def sigmoid(x: JaxArray, /, *, out: Optional[JaxArray] = None) -> JaxArray:
 
 
 def softmax(
-    x: JaxArray, /, *, axis: Optional[int] = None, out: Optional[JaxArray] = None
+    x: JaxArray, /, *, axis: Optional[int] = -1, out: Optional[JaxArray] = None
```

Comment:
`Optional` should be used when the default value of an argument is `None`, we can remove it from here. Similar changes apply to all other functions

Reply:
Woops, makes sense!